.github/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# Changes from release 2024/06 to 2024/07
+1. AIRAC (2407) - Update Jersey (JSY) VOR coordinates - thanks to @chssn
```

Comment:
```suggestion
1. AIRAC (2407) - Updated Jersey (JSY) VOR coordinates - thanks to @chssn
```
Sorry, didn't notice the first time, Changelog entries use the past tense of the verb, PR titles use the imperative. Per the [style guide](https://github.com/VATSIM-UK/UK-Sector-File/blob/main/.github/Style%20Guide.md#entry-summary-).